source/extensions/filters/http/common/factory_base.h

```diff
@@ -159,6 +159,23 @@ class DualFactoryBase : public CommonFactoryBase<ConfigProto, RouteConfigProto>,
   createFilterFactoryFromProtoTyped(const ConfigProto& proto_config,
                                     const std::string& stats_prefix, DualInfo info,
                                     Server::Configuration::ServerFactoryContext& context) PURE;
+
+  Envoy::Http::FilterFactoryCb createFilterFactoryFromProtoWithServerContext(
```

Comment:
This change will be removed when we merge with: https://github.com/envoyproxy/envoy/pull/33013